### ISSUE TITLE
kdb5_util: Cannot read password while reading master key from keyboard

### ISSUE THREAD

**Issue by JaBurd** [atomic-penguin/cookbook-krb5#44]
I was getting an error when trying to create-admin-principal in the kadmin_init recipe.  I walked back manually to troubleshoot and found the failure actually happens at the krb5 db creation step.  It appears kdb5_util doesn't like the echo of the passwords as manually typing them in and hitting enter does not result in the same error.

`Centos 7.5  -  3.10.0-862.14.4.el7.x86_64`

`Chef 14.5.33`

`Kerberos 5 version 1.15.1`

```
# echo 'password\npassword\n' | kdb5_util -r <default_realm create> -s
Loading random data
Initializing database '/var/kerberos/krb5kdc/principal' for realm '<default_realm>',
master key name 'K/M@<default_realm>'
You will be prompted for the database Master Password.
It is important that you NOT FORGET this password.
Enter KDC database master key:
Re-enter KDC database master key to verify:
kdb5_util: Cannot read password while reading master key from keyboard
```

**Comment by JaBurd** [comment]
It appears we can re-format this command to get a successful creation.

```
# { echo 'password\n'; echo 'password\n'; } | kdb5_util -r <default_realm> create -s
Loading random data
Initializing database '/var/kerberos/krb5kdc/principal' for realm '<default_realm>',
master key name 'K/M@<default_realm>'
You will be prompted for the database Master Password.
It is important that you NOT FORGET this password.
Enter KDC database master key:
Re-enter KDC database master key to verify:
```

**Comment by wolf31o2** [comment]
This is resolved via #45
